pkg/controller/installation/core_controller.go

```diff
@@ -590,6 +590,22 @@ func (r *ReconcileInstallation) Reconcile(request reconcile.Request) (reconcile.
 		return reconcile.Result{}, err
 	}
 
+	// save off the base install before merging overrides, so we can write it back to the 'default' installation later.
+	base := instance
+
+	// update Installation with 'overrides'
+	overrides := operator.Installation{}
+	if err := r.client.Get(ctx, utils.OverridesInstanceKey, &overrides); err != nil {
+		if !apierrors.IsNotFound(err) {
+			reqLogger.Error(err, "An error occurred when querying the 'overrides' Installation resource")
+			return reconcile.Result{}, err
+		}
+		reqLogger.V(5).Info("no 'overrides' installation found")
+	} else {
+		reqLogger.V(5).Info("merging overrides")
+		instance.Spec = overrideInstallationSpec(instance.Spec, overrides.Spec)
+	}
+
 	reqLogger.V(2).Info("Loaded config", "config", instance)
 
 	// Validate the configuration.
```

Comment:
I think we should validate pre-override and post so we can clearly indicate if the issue is with the base config or the overridden config.